epiline.R

```diff
@@ -0,0 +1,43 @@
+# Taken from https://github.com/BDI-pathogens/EpiLine/tree/main?tab=readme-ov-file#example-results
+
+library( EpiLine )
+set.seed( 1 )
+
+# Run the EpiLine example
+run <- function(arrow_table) {
```

Comment:
parameter seems unused. weirdly, it runs fine for me whether I provide it or not. this is strange behavior it me. remove?